packages/ui/src/elements/ListDrawer/DrawerContent.tsx

```diff
@@ -59,15 +59,6 @@ export const ListDrawerContent: React.FC<ListDrawerProps> = ({
       collectionSlug: selectedOption.value,
     })
 
-  useEffect(() => {
-    if (selectedCollectionFromProps && selectedCollectionFromProps !== selectedOption?.value) {
-      setSelectedOption({
-        label: collections.find(({ slug }) => slug === selectedCollectionFromProps).labels,
-        value: selectedCollectionFromProps,
-      })
-    }
-  }, [selectedCollectionFromProps, collections, selectedOption])
-
```

Comment:
I'm not sure what the purpose of this useEfffect was, since `selectedCollectionFromProps` and `selectedOption` are effectively the same property used in the drawer context.

I tried to see if disabling it broke anything, but I didn't find any strange behavior.